docs/docfx.json

```diff
@@ -1274,46 +1276,90 @@
       "src": "breadcrumb",
       "dest": "office_js_breadcrumb"
     },
-    
+	
     {
       "files": [
         "**/*.md",
         "**/*.yml"
       ],
-      "group": "powerpoint-js-1.1",
-      "src": "docs-ref-autogen/powerpoint_1_1",
+      "group": "powerpoint-js-1.2",
+      "src": "docs-ref-autogen/powerpoint_1_2",
```

Comment:
This (and the other 4 entries) should stay 1.1. You need all these for each and every moniker.